src/Components/User.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CommentList from './CommentList';
+
+class User extends React.Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components